src/sass/components/_path.scss

```diff
@@ -0,0 +1,23 @@
+.slds-path__item.status-path__item {
+  min-width: auto;
+  &.slds-is-current.slds-is-active {
+    background-color: $brand-background-primary;
+  }
+}
+
+.slds-path__nav .slds-is-active {
+  &:before,
+  &:after {
+    background-color: $brand-background-primary;
+  }
+  &:first-child {
+    &:before,
+    &:after {
+      background: $brand-background-primary;
+    }
+  }
+}
+
+.slds-text-longform ul {
+  list-style: none;
+}
```

Comment:
If we're overriding native SLDS styles, I think we should namespace to our particular implementation of the component.

Reply:
@stacyk I added a namespaced class, and also tried a descendant selector to change the bullet points here. Any ideas on the best way to override this style?